externs/es3.js

```diff
@@ -935,6 +935,14 @@ Boolean.prototype.toSource = function() {};
 Boolean.prototype.toString = function() {};
 
 /**
+ * @return {boolean}
+ * @nosideeffects
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/valueOf
+ * @override
```

Comment:
It looks like Boolean actually extends Object, because the JSTypeRegistry sets it up to be so, even though it doesn't say it explicitly in the externs file. This also reflects reality: http://jsconsole.com/?(new%20Boolean)%20instanceof%20Object

So I think the `@override` is correct.